q2_diversity/_alpha/alpha_rarefaction_assets/src/init.js

```diff
@@ -0,0 +1,58 @@
+/* global categories */
+/* global metrics */
+import { select } from 'd3';
+import state from './state';
+import { addMetricPicker, addCategoryPicker } from './toolbar';
+
+export default function init() {
+  const metric = metrics[0];
+  const category = categories[0];
+  const body = select('#main');
+  // Order matters here.
+  body.append('h1').text('Alpha Rarefaction');
+  const controlsRow = body.append('div').attr('class', 'controls row');
+  const vizRow = body.append('div').attr('class', 'viz row');
+
+  const plotCol = vizRow.append('div')
+                    .attr('class', 'col-lg-9')
+                    .style('width', '1120px');
+  const legendCol = vizRow.append('div')
+                      .attr('class', 'col-lg-2')
+                      .style('height', '1000px');
```

Comment:
Could these be defined by the template instead? It would be a bit more readable, and the d3 only handles the svg nodes.